modules/client-configuration/pages/supported-features.adoc

```diff
@@ -72,6 +72,12 @@ ifeval::[{suma-content} == true]
 | {cross}
 | {check}
 
+| {opensuse} Leap 16
+| {check}
```

Comment:
As I mentioned above, as of today the four architectures are enabled, but check later to see if that's still the case.